### ISSUE TITLE
VAPI "from" and "to" fields are incorrectly identical

### ISSUE THREAD

**Issue by lornajane** [Nexmo/nexmo-developer#985]
## Description

The documentation shows that to make a call, the `from` and `to` fields must each contain a phone number, sip endpoint or websocket. The fields are defined identically in the OAS and therefore render the same. However this isn't accurate.  The `to` is an array of those things, and the `from` is only one of them.

For example, this is the curl building block for making a call:

```
curl -X POST https://api.nexmo.com/v1/calls\
  -H "Authorization: Bearer "$JWT\
  -H "Content-Type: application/json"\
  -d '{"to":[{"type": "phone","number": "'$TO_NUMBER'"}],
      "from": {"type": "phone","number": "'$NEXMO_NUMBER'"},
      "answer_url":["https://raw.githubusercontent.com/nexmo-community/ncco-examples/gh-pages/text-to-speech.json"]}'
```
It's subtle, but the `from` is correctly `OneOf` these things and the `to` should be an array containing any number of these things.

## Related resources

https://developer.nexmo.com/api/voice#createCall

**Comment by mheap** [comment]
I've had a little play with this and it's not as easy as it seems. Using the Pet Store example, the following rendering works:

```
paths:
  /pets:
    post:
      description: Creates a new pet in the store. Duplicates are allowed
      requestBody:
        description: Pet to add to the store
        required: true
        content:
          application/json:
            schema:
              properties:
                name:
                  $ref: "#/components/schemas/RealName"
```

But updating a property to be `oneOf` doesn't:

```
paths:
  /pets:
    post:
      description: Creates a new pet in the store.Duplicates are allowed
      requestBody:
        description: Pet to add to the store
        required: true
        content:
          application/json:
            schema:
              properties:
                name:
                  oneOf: 
                    $ref: "#/components/schemas/RealName"
                    $ref: "#/components/schemas/NickName"
```

I think we may need to use a `oneOf` on the whole schema, which will make the docs much larger as the entire schema will be repeated three times. This will take some concerted effort to work through I think

**Comment by mheap** [comment]
Solved in 5bdcece7df5eaa858ce74480396c1da6f9ec313f and clarified in febd04eef7be431cfcaf9ab558cfa5f28df996f8
